homeassistant/util/package.py

```diff
@@ -18,6 +18,11 @@
 
 def is_virtual_env() -> bool:
     """Return if we run in a virtual environment."""
+    # Check if in a conda env
+    env = os.environ.copy()
+    if env.get("CONDA_SHLVL", "0") != "0":
```

Comment:
```suggestion
    if os.environ.get("CONDA_SHLVL"):
```
Isn't this the same thing?

Reply:
Hi @Kane610, thanks for the comment.  It's actually not quite the same, `CONDA_SHLVL` can actually be "0", or it could not be present, so the default is not always hit. 

On a user account that does not have conda installed: 
`os.environ.get("CONDA_SHLVL")` is `None`

On a user that has conda installed but has deactivated the base environment (for example, from running `conda deactivate` after first logging in):
`os.environ.get("CONDA_SHLVL")` is `"0"`

On a user that has conda installed and has the base environment active (the default when first logging in):
`os.environ.get("CONDA_SHLVL")` is `"1"`

On a user that has conda installed and has activated an environment on top of the base environment (for example, with `conda activate myenv`: 
`os.environ.get("CONDA_SHLVL")` is `"2"`

With conda, you can continue to activate more environments on top of each other, and `CONDA_SHLVL` will continue to increment to > 2, though this can result in unexpected package resolution and is not recommended.

Reply:
```suggestion
    env = os.environ.copy()
    if env.get("CONDA_SHLVL") and env.get("CONDA_SHLVL") != "0":
```
Maybe this is more explicit given that explanation?